docs/Contributing/API-for-contributors.md

```diff
@@ -596,6 +596,7 @@ Once base64-decoded, they are PEM-encoded certificate and keys.
 | team_name | string | query | _Available in Fleet Premium_ The name of the team to apply the custom settings to. Only one of `team_name`/`team_id` can be provided. |
 | dry_run   | bool   | query | Validate the provided profiles and return any validation errors, but do not apply the changes.                                    |
 | profiles  | json   | body  | An array of strings, the base64-encoded .mobileconfig (macOS) or XML (Windows) files to apply.                                        |
+| labels   | array   | query | _Available in Fleet Premium_ An array of labels to filter hosts in a team (or no team) that should get a profile. |
```

Comment:
@marko-lisica and @roperzh is this right? Or do we need to update `profiles` so that it accepts an array of objects instead of an array of strings.

Each object in `profiles` has a `profile` key that accepts the base64-encoded .mobileconfig and `labels` which accepts an array of label names.

That way, each profile has a unique set of labels.

Reply:
That's right. This is a mistake.